internal/preview/preview.go

```
@@ -69,7 +68,8 @@ func Create(params CreateParams) error {
 	text, err := func() ([]byte, error) {
 		tree := yml.Clone(source.File.Tree)
 		yml.SetOrAddNodeValue(tree, "metadata.name", target)
-		yml.SetOrAddNodeValue(tree, "metadata.labels."+strings.ReplaceAll(v1alpha1.PreviewLabel, ".", "\\."), "true")
+		yml.SetOrAddNodeValue(tree, "metadata.labels."+yml.EscapePathSegment(v1alpha1.PreviewLabel), "true")
```

golangci-lint (2.12.2)

[error] Error return value of `yml.SetOrAddNodeValue` is not checked

(errcheck)